src/utils/apub_name.rs

```diff
@@ -14,14 +13,17 @@ fn format_apub_name<const PREFIX: char>(name: &str, instance: &str) -> String {
 }
 
 fn create_apub_name<const PREFIX: char>(name: &str, actor_id: &str) -> String {
+  // TODO Strange issue where the must be defined, or leptos will fail at startup
+  let default_url = "https://example.com";
```

Comment:
I started getting leptos startup errors all the sudden without this.

Reply:
I made a slightly different fix for this issue in my fluent branch that I will be pushing up soon. I'll see if you like it more than this one.